### PR TITLE
Test normalize ast

### PR DESCRIPTION
**Pull request by wert-rar**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -8,6 +8,7 @@ __pycache__/
 
 # Distribution / packaging
 .Python
+
```
Comment (0dminnimda) [inline .gitignore:11]: Why did you add the line here?

File: .gitignore [hunk 2]
```diff
@@ -94,7 +95,7 @@ venv/
 ENV/
 env.bak/
 venv.bak/
-
+.idea/
```
Comment (0dminnimda) [inline .gitignore:98]: This is for IntelliJ IDEA as I understand it, so let's add a comment about that.

As you can see, all other sections are formatted, i.e. separated by newline and with a comment, like
```
# Rope project settings
.ropeproject
```

File: tests/test_normalize_ast.py [hunk 3]
```diff
@@ -81,8 +81,7 @@ def test_visit_ExtSlice():
 
 
 ############ normalize_ast #############
-# TODO: add a test that shows that ast.fix_missing_locations is in use,
-# it is probably better to take tests for ast.fix_missing_locations from python sources.
+
```
Comment (0dminnimda) [inline tests/test_normalize_ast.py:84]:
```suggestion
```

All other similar comments are formatted like this:
```python
<code>


### <comment> ###

<code>
```
so let's stick with it.

File: pynalyser/normalize_ast.py [hunk 4]
```diff
@@ -31,4 +31,4 @@ def visit_Constant(self, node: ast.Constant) -> ast.Constant:
 
 
 def normalize_ast(tree: ast.AST) -> ast.AST:
-    return ast.fix_missing_locations(AstNormalizer().visit(tree))
+    return AstNormalizer().visit(tree)
```
Comment (0dminnimda) [inline pynalyser/normalize_ast.py:34]: What is the reason for removing this function?

File: .gitignore (hunk 2 above)
Comment (wert-rar) [reply .gitignore:98]: ok i will do it

File: pynalyser/normalize_ast.py (hunk 4 above)
Comment (wert-rar) [reply pynalyser/normalize_ast.py:34]: you asked me to do this...

File: .gitignore (hunk 1 above)
Comment (0dminnimda) [reply .gitignore:11]: More precisely, I think you should remove this since there is no point in this change.
```suggestion
```

File: tests/test_normalize_ast.py (hunk 3 above)
Comment (wert-rar) [reply tests/test_normalize_ast.py:84]: ok i will stick with it!

File: pynalyser/normalize_ast.py (hunk 4 above)
Comment (0dminnimda) [reply pynalyser/normalize_ast.py:34]: hmph ok

File: tests/test_normalize_ast.py (hunk 3 above)
Comment (0dminnimda) [reply tests/test_normalize_ast.py:84]: You can click on the "commit suggestion" button to satisfy the above format.

**Comment by 0dminnimda** [comment]
Congratulations 🥳

**Comment by 0dminnimda** [comment]
@all-contributors let's try adding @wert-rar as a test fixer ;]

**Comment by 0dminnimda** [comment]
@all-contributors let's add @wert-rar for test
